lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -207,6 +240,12 @@ def verify_userauth(id_token, group_name=None, internal_token_only=False):
         valid_issuer = False
         keyset = None
         target_client_ids = None
+        checkAud = True
+
+        AUTH_PUBKEYS = os.getenv('AUTH_PUBKEYS', '').strip()
```

Comment:
We should be loading this from the Core BB not an env var. This was implemented previously on a different branch.